static/js/click_handlers.js

```diff
@@ -499,7 +499,7 @@ export function initialize() {
         narrow.by_recipient(row_id, {trigger: "message header"});
     });
 
-    $("#message_feed_container").on("click", ".narrows_by_topic", function (e) {
+    $("#message_feed_container").on("click", "div.narrows_by_topic", function (e) {
```

Comment:
This is the other thing we added to fix this bug (which also means we don't have to delete the other code if we don' want to).

Reply:
Hmm, I'm pretty sure the intent of this block of code is to make clicking any element with the `narrows_by_topic` class call `narrow.by_topic(row_id, {trigger: "message header"});` -- I'm 99% confident the intent is to just to add the `trigger` bonus parameter, which is intended for instrumentation but also may be used for minor functional differences. So it's probably not helpful to make this change -- it'll effectively stop passing that parameter unexpectedly.

So I suspect we should drop this hunk.